radio/src/dataconstants.h

```diff
@@ -649,6 +650,17 @@ enum MixSources {
   MIXSRC_GYRO2,                         LUA_EXPORT("gyry", "Gyro Y")
 #endif
 
+#if defined(SPACEMOUSE)
+  MIXSRC_FIRST_SPACEMOUSE SKIP,
+  MIXSRC_SPACEMOUSE_1 = MIXSRC_FIRST_SPACEMOUSE, LUA_EXPORT("smA", "SpaceMouse A")
+  MIXSRC_SPACEMOUSE_2, LUA_EXPORT("smB", "SpaceMouse B")
+  MIXSRC_SPACEMOUSE_3, LUA_EXPORT("smC", "SpaceMouse C")
+  MIXSRC_SPACEMOUSE_4, LUA_EXPORT("smD", "SpaceMouse D")
+  MIXSRC_SPACEMOUSE_5, LUA_EXPORT("smE", "SpaceMouse E")
+  MIXSRC_SPACEMOUSE_6, LUA_EXPORT("smF", "SpaceMouse F")
+  MIXSRC_LAST_SPACEMOUSE SKIP = MIXSRC_SPACEMOUSE_6,
```

Comment:
In UI smA - smF so why not SPACEMOUSE_A - SMARTMOUSE_F and same in yaml below?

Reply:
Thx for the tip. Changed in latest commit uniformly to SPACEMOUSE_A to *_F.